src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -588,6 +587,29 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
                         "--namespace", "{}".format(consts.Release_Install_Namespace),
                         "--create-namespace",
                         "--output", "json"]
+
+    # Special configurations from 2022-09-01 ARM metadata.
+    metadata = get_metadata(resource_manager, "2022-09-01")
```

Comment:
this same call is made in the function 'get_config_dp_endpoint' in the 'connectedk8s connect' command as well. is it possible to optimize and make this call only once?